packages/svelte-vscode/package.json

```diff
@@ -23,7 +23,7 @@
         "url": "https://github.com/sveltejs/language-tools/issues"
     },
     "homepage": "https://github.com/sveltejs/language-tools#readme",
-    "displayName": "Svelte Beta",
+    "displayName": "Svelte for VS Code",
```

Comment:
Bikeshedding: What do you think about naming it `Svelte (Official)`?

Reply:
Not a fan personally, it feels a bit like calling yourself cool

Given the birtle's one will be deprecated it doesn't really need to stand out in that way

Reply:
Fair enough. After Birtle's is deprecated/renamed/removed, we could think about renaming it to just "Svelte"?

Reply:
Legit